source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,122 @@
+
+During Sync 
+~~~~~~~~~~~
+
+The :ref:`start <c2c-api-start>` endpoint begins the 
+:ref:`RUNNING <c2c-state-running>` state. 
+
+During the sync, ``mongosync`` syncs data from the source cluster to the 
+destination cluster. If you make subsequent writes to the source cluster, 
+``mongosync`` applies the operations to the destination cluster.  
+
+For every collection on the source, ``mongosync`` creates a corresponding 
+collection on the destination. 
+
+``mongosync`` temporarily alters some collection characteristics during 
+the ``RUNNING`` state and changes these back to their source 
+cluster characteristics during the ``COMMIT`` state. For more information, see
+:ref:`Commit Alters <c2c-commit-alters>`.
+
+Collection Copy
+'''''''''''''''
+
+Collection copy is the initial full data transfer that copies each collection 
+from the source cluster to the destination cluster. ``mongosync`` performs 
+collection copy by splitting each source collection into partitions and copying
+all the partitions to their corresponding destination collections.
+
+Change Event Application
+''''''''''''''''''''''''
+
+The change event application (CEA) process continuously updates 
+the destination cluster when you make changes to the source cluster. If there 
+are multiple ``mongosync`` instances, all the ``mongosync`` instances finish 
+collection copy before starting the CEA process.
+
+During CEA, ``mongosync`` receives :ref:`change stream <changeStreams>` events 
+from the source cluster, converts these events into writes, and applies these 
+writes on the destination cluster. CEA ensures that :ref:`CRUD <crud>` and 
+:ref:`DDL <c2c-mongosync-ddl>` events from the source cluster are accurately
+applied to the destination cluster.
+
+Pausing Sync
+~~~~~~~~~~~~
+
+The :ref:`pause <c2c-api-pause>` endpoint begins the 
+:ref:`PAUSED <c2c-state-paused>` state.
+
+During the ``PAUSED`` state, the sync process is paused.
+
+The :ref:`resume <c2c-api-resume>` endpoint resumes a paused synchronization 
+session and returns ``mongosync`` to the ``RUNNING`` state. 
+
+Finalizing Sync
+~~~~~~~~~~~~~~~
+
+The :ref:`commit <c2c-api-commit>` endpoint begins the 
+:ref:`COMMITTING <c2c-state-committing>` state.
+
+During the ``COMMITTING`` state, ``mongosync`` finalizes the sync between the 
+source cluster and the destination cluster by stopping continuous sync 
+between them. It also restores collection characteristics that ``mongosync`` 
+temporarily alters during synchronization. 
+
+Once you call the ``commit`` endpoint, ``mongosync`` halts any writes on the 
+source cluster and retrieves the timestamp of the most recent operation in the 
+source cluster. 
+
+If there are no errors, ``mongosync`` sends a success message and exits CEA by 
+processing events up until the commit timestamp. Then, ``mongosync`` corrects 
+indexes and resolves document conflicts. If write blocking is enabled, 
+``mongosync`` turns write operations back on. 
```

Comment:
I struggled with this because in most situations it is mongosync that enables write blocking, not the user (above I write "``mongosync`` halts any writes on the 
source cluster"). What do you think about saying, "If write blocking is ``true``"?